src/Specular/Internal/Incremental/Optional.purs

```diff
@@ -0,0 +1,27 @@
+module Specular.Internal.Incremental.Optional
```

Comment:
Cool thing, bringing nullability back ;) I would even name it `Nullable` to make people beware ;)

Reply:
Note: PureScript already has [Nullable](https://pursuit.purescript.org/packages/purescript-nullable/4.1.1/docs/Data.Nullable#t:Nullable). `Optional` is different, because it can contain types which can be represented by JavaScript `null`, for example `Foreign`.